pkgs/top-level/php-packages.nix

```diff
@@ -49,12 +49,19 @@
 }:
 
 lib.makeScope pkgs.newScope (self: with self; {
-  buildPecl = import ../build-support/build-pecl.nix {
+  buildPecl = import ../build-support/php/build-pecl.nix {
     php = php.unwrapped;
     inherit lib;
     inherit (pkgs) stdenv autoreconfHook fetchurl re2c nix-update-script;
   };
 
+  buildPHPproject = import ../build-support/php/build-php.nix {
```

Comment:
Might be nicer to have `buildComposerVendorDir` or similar helper. Derivation builders are not really composable.

Reply:
I guess I can consider this as fixed now!!!